tests/jballerina-integration-test/src/test/resources/configurables/configUnionTypesProject/configUnionTypes/main.bal

```diff
@@ -32,6 +32,17 @@ type HttpResponse record {|
 configurable HttpResponse httpResponse = ?;
 configurable mod1:Country country = ?;
 
+configurable anydata anydataVar = ?;
+configurable int|string intStringVar = 2;
+configurable anydata[] anydataArray = ?;
+configurable map<anydata> anydataMap = ?;
+configurable table<map<anydata>> anydataTable = ?;
+
+configurable configLib:One|configLib:Two|configLib:Three number = ?;
+configurable configLib:GrantConfig config1 = ?;
+configurable configLib:GrantConfig config2 = ?;
+configurable configLib:GrantConfig config3 = ?;
```

Comment:
Shall we have tests with union of maps, table (with/without keys)?

Reply:
Added unions of map and map of union records. but adding tests for unions of table cause issue fixed in https://github.com/ballerina-platform/ballerina-lang/pull/31205. So I'll add test with a separate PR

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/31222